src/main/java/org/codelibs/fess/app/web/admin/scheduledjob/AdminScheduledjobAction.java

```diff
@@ -199,113 +239,139 @@ public HtmlResponse confirmfromupdate(final ScheduledjobEditForm form) {
         return redirect(getClass());
     }
 
     @Execute(token = TxToken.VALIDATE)
-    public HtmlResponse update(final ScheduledjobEditForm form) {
+    public HtmlResponse update(final EditForm form) {
+        verifyCrudMode(form.crudMode, CrudMode.EDIT);
         validate(form, messages -> {}, toEditHtml());
-        scheduledJobService.store(createScheduledJob(form));
-        saveInfo(messages -> messages.addSuccessCrudUpdateCrudTable(GLOBAL));
+        createScheduledJob(form).ifPresent(entity -> {
+            copyBeanToBean(form, entity, op -> op.exclude(Constants.COMMON_CONVERSION_RULE));
+            scheduledJobService.store(entity);
+            saveInfo(messages -> messages.addSuccessCrudUpdateCrudTable(GLOBAL));
+        }).orElse(() -> {
+            throwValidationError(messages -> messages.addErrorsCrudCouldNotFindCrudTable(GLOBAL, form.id), toEditHtml());
+        });
         return redirect(getClass());
     }
 
     @Execute
-    public HtmlResponse delete(final ScheduledjobEditForm form) {
-        verifyCrudMode(form, CrudMode.DELETE);
-        scheduledJobService.delete(getScheduledJob(form));
-        saveInfo(messages -> messages.addSuccessCrudDeleteCrudTable(GLOBAL));
+    public HtmlResponse delete(final EditForm form) {
+        verifyCrudMode(form.crudMode, CrudMode.DELETE);
+        validate(form, messages -> {}, toEditHtml());
+        final String id = form.id;
+        scheduledJobService.getScheduledJob(id).ifPresent(entity -> {
+            scheduledJobService.delete(entity);
+            saveInfo(messages -> messages.addSuccessCrudDeleteCrudTable(GLOBAL));
+        }).orElse(() -> {
+            throwValidationError(messages -> messages.addErrorsCrudCouldNotFindCrudTable(GLOBAL, id), toEditHtml());
+        });
         return redirect(getClass());
     }
 
     @Execute
-    public HtmlResponse start(final ScheduledjobEditForm form) {
-        verifyCrudMode(form, CrudMode.CONFIRM);
-        final ScheduledJob scheduledJob = getScheduledJob(form);
-        try {
-            scheduledJob.start();
-            saveInfo(messages -> messages.addSuccessJobStarted(GLOBAL, scheduledJob.getName()));
-        } catch (final Exception e) {
-            throwValidationError(messages -> {
-                messages.addErrorsFailedToStartJob(GLOBAL, scheduledJob.getName());
-            }, toEditHtml());
-        }
+    public HtmlResponse start(final EditForm form) {
+        final String id = form.id;
+        scheduledJobService.getScheduledJob(id).ifPresent(entity -> {
+            try {
+                entity.start();
+                saveInfo(messages -> messages.addSuccessJobStarted(GLOBAL, entity.getName()));
+            } catch (final Exception e) {
+                throwValidationError(messages -> {
+                    messages.addErrorsFailedToStartJob(GLOBAL, entity.getName());
+                }, toEditHtml());
+            }
+        }).orElse(() -> {
+            // TODO 
+            });
```

Comment:
This "OrElse" is maybe unreachable. (but I'm not sure).
I couldn't find a suitable error message.